app/controllers/api/v2/config_reports_controller.rb

```diff
@@ -73,7 +73,7 @@ def setup_search_options
 
       def resource_scope(options = {})
         options[:permission] = :view_config_reports
-        super(options).my_reports
+        super(options).my_reports.includes(:messages, :sources)
```

Comment:
I could not find an example how to do that, do I simply do `ConfigReport.includes().find()` instead? I though I must use `resource_scope` because of authorization.

Reply:
resouce_scope still returns the scope, you can do something like `resource_scope.includes().find()`

Reply:
Ok thanks, how about this?